### PR TITLE
Chore(cypress): Review and Fix tests failing on master

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Jira : https://mattermost.atlassian.net/browse/MM-58115

logged: 
- https://mattermost.atlassian.net/browse/MM-58114
- https://mattermost.atlassian.net/browse/MM-58112

```release-note
NONE
```

### CODE REVIEW

File: e2e-tests/cypress/tests/integration/channels/archived_channel/post_menu_spec.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe('Archived channels', () => {
 
     it('MM-T1721 Archive channel posts menu should have copy link and reply options', () => {
         // # Click to add a channel description
-        cy.findByRoleExtended('button', {name: 'Add a channel header'}).should('be.visible').click();
+        cy.findByRoleExtended('button', {name: 'Set header'}).should('be.visible').click();
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/archived_channel/post_menu_spec.ts:29]: This still failed.

File: e2e-tests/cypress/tests/integration/channels/integrations/integrations_spec.js [hunk 2]
```diff
@@ -445,5 +445,5 @@ describe('Integrations page', () => {
 
 function integrationPageTitleIsBold(title) {
     cy.get('.section-title__text').contains(title).click();
-    cy.get('.item-details__name').should('be.visible').and('have.css', 'font-weight', '400');
+    cy.get('.item-details__name').should('be.visible').and('have.css', 'font-weight', '600');
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/integrations/integrations_spec.js:448]: One failed on MM-T570

File: e2e-tests/cypress/tests/integration/channels/markdown/markdown_text_spec.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ describe('Markdown message', () => {
     testCases.forEach((testCase) => {
         it(testCase.name, () => {
             // #  Post markdown message
-            cy.postMessageFromFile(`markdown/${testCase.fileKey}.md`).wait(TIMEOUTS.FIVE_SEC);
+            cy.postMessageFromFile(`markdown/${testCase.fileKey}.md`).wait(TIMEOUTS.TWO_SEC);
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/markdown/markdown_text_spec.js:52]: Other 3 failed.

File: e2e-tests/cypress/tests/integration/channels/team_settings/closed_team_invite_by_email_spec.js [hunk 4]
```diff
@@ -123,7 +123,7 @@ describe('Team Settings', () => {
         // * Check that the display name of the team the user was invited to is being correctly displayed
         cy.uiGetLHSHeader().findByText(testTeam.display_name);
 
-        // * Check that the 'Beginning of Town Square' message is visible
-        cy.findByText('Beginning of Town Square').should('be.visible');
+        // * Check that the 'Town Square' channel title is visible
+        cy.get('h2.channel-intro__title').should('be.visible').should('have.text', 'Town Square');
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/team_settings/closed_team_invite_by_email_spec.js:127]: This failed.

File: e2e-tests/cypress/tests/integration/channels/toast/new_messages_toast_spec.js [hunk 5]
```diff
@@ -138,7 +138,7 @@ describe('Toast', () => {
             cy.uiClickSidebarItem(testChannelName);
 
             // # Scroll up so bottom is not visible
-            scrollUp();
+            scrollUpAndPostAMessage(otherUser, testChannelId, oldPostNumber);
 
             // # New message toast is present with expected number of messages
             cy.get('div.toast').should('be.visible');
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/toast/new_messages_toast_spec.js:144]: MM-T1786 failed

**Review by saturninoabril — APPROVED** [review]
Wow, great to see all of the fixes! Some still failed but non-blocking and can be addressed separately.

File: e2e-tests/cypress/tests/integration/channels/archived_channel/post_menu_spec.ts (hunk 1 above)
Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/archived_channel/post_menu_spec.ts:29]: image

File: e2e-tests/cypress/tests/integration/channels/toast/new_messages_toast_spec.js (hunk 5 above)
Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/toast/new_messages_toast_spec.js:144]: this passes locally, earlier it used to fail in local as well.
image

File: e2e-tests/cypress/tests/integration/channels/markdown/markdown_text_spec.js (hunk 3 above)
Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/markdown/markdown_text_spec.js:52]: @saturninoabril 
I need to make be ask Dev to fix this. In those 3 cases, When mark down text is copied there's `\n` coming up.
and not <br>. I'll check with devs once.

File: e2e-tests/cypress/tests/integration/channels/integrations/integrations_spec.js (hunk 2 above)
Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/integrations/integrations_spec.js:448]: fixed

**Review by harshilsharma63 — APPROVED** [review]
LGTM!

**Pull request merged**